docker/quick-start/Dockerfile

```diff
@@ -0,0 +1,27 @@
+
+# this container should match the manual steps in 
+# https://docs.bacalhau.org/running-node/quick-start
+
+# the idea of the quickstart is to help someone bring up the simplest global cluster node
+
+# TODO: build-args for what version of bacalhau?
+
+FROM ubuntu:22.04 as bacalhau
+
+# crib from, or use https://github.com/filecoin-project/bacalhau/blob/b2120012ea/ops/terraform/remote_files/scripts/install-node.sh
+RUN apt-get update -y && apt-get install -y \
+      ca-certificates \
+      curl \
+      gnupg \
+      lsb-release
+
+RUN curl -sL https://get.bacalhau.org/install.sh | bash
+
+RUN cd /tmp \
```

Comment:
Why are we installing docker here? I thought we were going to leverage the host's docker? (I'll double check the other comments in a second)

Reply:
this is installing the docker commandline - I didn't check, I presumed that `bacalhau` exec's `docker`? (if not YAY)

Reply:
It looks like most docker things are done via the APIClient, but there's still `https://github.com/filecoin-project/bacalhau/blob/main/pkg/executor/docker/executor.go#L287`